External/Plugins/ASCompletion/Completion/TemplateUtils.cs

```diff
@@ -45,18 +45,22 @@ public static string GetModifiers(MemberModel member)
 
         public static string ToDeclarationWithModifiersString(MemberModel m, string template)
         {
-            bool isConstructor = (m.Flags & FlagType.Constructor) > 0;
+            var accessModifier = GetModifiers(m).Trim();
+            if (accessModifier == "private" && ASContext.Context.Features.methodModifierDefault == Visibility.Private
```

Comment:
I think this could be more generic, but I see it was already like this already in the past.